### DISCUSSION TITLE
[@types/firefox-webext-browser] 2 missing functions in `browser.bookmarks`

### DISCUSSION THREAD

**Discussion by tametsi** [DefinitelyTyped/DefinitelyTyped#57890] [Issues with a @types package] [answered]
somewhere here
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/543e123b3912be712c402cb356e7b8d8f80fdc40/types/firefox-webext-browser/index.d.ts#L5953-L6006
are 2 function declarations missing:

```typescript
bookmarks.remove()
// and
bookmarks.removeTree()
```

According to [MDN](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/bookmarks) they should be available

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment, marked as answer]
@Vis17 PR is ready #57928
